src/actions/HelpDropdownActions.ts

```diff
@@ -0,0 +1,52 @@
+import { createAction } from 'typesafe-actions';
+import * as API from '../services/Api';
+import { KialiProduct } from '../store/Store';
+import { MessageType } from '../types/MessageCenter';
+import { MessageCenterActions } from './MessageCenterActions';
+
+export enum HelpDropdownActionKeys {
+  STATUS_REFRESH = 'STATUS_REFERSH'
+}
+
+// synchronous action creators
+export const HelpDropdownActions = {
+  statusRefresh: createAction(
+    HelpDropdownActionKeys.STATUS_REFRESH,
+    (status: { [key: string]: string }, products: KialiProduct[], warningMessages: string[]) => ({
+      type: HelpDropdownActionKeys.STATUS_REFRESH,
+      status: status,
+      products: products,
```

Comment:
product? Kiali is a project, not a product :)
product is something commercial

Reply:
^^ business skills++